src/zcash/util.cpp

```diff
@@ -10,3 +11,35 @@ std::vector<unsigned char> convertIntToVectorLE(const uint64_t val_int) {
 
     return bytes;
 }
+
+// Convert bytes into boolean vector.
```

Comment:
Add a comment that the bits in each byte are encoded from most-significant to least significant.